.github/workflows/main.yml

```diff
@@ -4,45 +4,36 @@ name: CI
 
 # Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the master branch
   push:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
     runs-on: ubuntu-latest
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
-      
+      - uses: actions/checkout@v4
+
       - name: setup-docker
-        uses: docker-practice/actions-setup-docker@51fd5009666ea5e3382fe125abf5d73d31386b59
+        uses: docker/setup-docker-action@v4
         with:
-          # Docker Version
-          docker_version: 20.10.11
-          # Docker Channel
-          docker_channel: stable
+          channel: stable
           
       - name: Setup Node.js environment
-        uses: actions/setup-node@v2.5.0
+        uses: actions/setup-node@v4
         with:
-          # Version Spec of the version to use.  Examples: 12.x, 10.15.1, >=10.15.0
-          node-version: 16.13
-          # Used to specify a package manager for caching in the default directory. Supported values: npm, yarn, pnpm
+          node-version: 22.x
           cache: npm
-          # Used to specify the path to a dependency file: package-lock.json, yarn.lock, etc. Supports wildcards or a list of file names for caching multiple dependencies.
           cache-dependency-path: package-lock.json
 
-      # Runs a single command using the runners shell
       - name: Install
         run: npm ci
+      
+      - name: Build
+        run: npm run build
```

Comment:
npm run test does already this step, doesn't?

https://github.com/YasharF/mailchecker/blob/master/package.json#L17

Reply:
Fixed
Test run: https://github.com/YasharF/mailchecker/actions/runs/13056260555/workflow